backend-docker/pier/db/db.go

```diff
@@ -0,0 +1,551 @@
+package db
+
+import (
+	"bytes"
```

Comment:
please use goimports for nice formatting: https://godoc.org/golang.org/x/tools/cmd/goimports